.gitignore

```diff
@@ -1,5 +1,147 @@
 .DS_Store
```

Comment:
Depending on the files created, we might want to have `.dockerignore` file too (with similar contents to dockerignore).
By default when the container is created according to Dockerfile, all of the files in the directory are transferred into local build context. Some of the files/directories could be actually useless - i.e. in many projects virtualenv that contains 3rd party binaries/sources which are re-downloaded anyways.